mx.jruby/suite.py

```diff
@@ -23,14 +23,12 @@
 
     "jruby-truffle" : {
       "subDir" : "",
+      "class" : "MavenProject",
```

Comment:
Ok, for the distribution i'll add support for the `class` attribute in mx for distribution and then you can do the same an create something that looks like a `JARDistribution` that just copies the jar created by maven. I'll have a quick to see if it would help to make an abstract `JARDistribution` that you can extend that has all the "consumption" aspects of `JARDistribution` but none of the "creation" aspects.